.github/workflows/check-and-test.yml

```diff
@@ -92,7 +92,6 @@ jobs:
           rustup toolchain install ${{ needs.get-version.outputs.toolchain }}
           rustup target add wasm32-unknown-unknown --toolchain ${{ needs.get-version.outputs.toolchain }}
           rustup component add rust-src
-          cargo update
```

Comment:
now I've looked at what this does we should not have done this. The solution was to upgrade the deep deps

Reply:
so this will be done in a different ticket?